deploy.js

```diff
@@ -20,25 +28,32 @@ const secrets = [
 ];
 
 async function main() {
+  let commands;
+  commands = ['yarn test'];
+
   if (BRANCH !== 'master') {
-    console.info('Not on master branch. Nothing to do.');
-    process.exit(0);
+    console.info('Skipping deploying on non-master branches');
+  } else if (isPullRequest === true) {
+    console.info('Skipping deployment commands in PRs');
+  } else {
+    commands = [
```

Comment:
This is fine, but you can also do

```js
commands.push(
  item1,
  item2,
  item3,
)
```

Reply:
I've been writing JS for years, and I didn't know that `array.push` accepts multiple arguments 😅 
Anyway, I still prefer the spread syntax.